contributors/chairs-and-techleads/README.md

```diff
@@ -85,8 +85,8 @@ Try #shoutouts on the Kubernetes slack!
       given the appropriate level of priority.
 #### Awards
 There are a few times a year that Kubernetes and the larger CNCF community 
-intentionally recognizes members of the commuity for certain roles and 
-accomplishments. The leads@ mailing list and slack channel will have 
+intentionally recognizes members of the community for certain roles and 
+accomplishments. The leads@ mailing list and the slack channel will have 
```

Comment:
```suggestion
accomplishments. The leads@ mailing list and slack channel will have 
```

The second "the" isn't desirable.  In this clause "leads@" modifies both "mailing list" and "slack channel", so putting a second "the" there breaks that pairing.

Reply:
suggestions applied